pkgs/misc/lguf-brightness/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, libusb, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "lguf-brightness";
+
+  version = "d194272b";
```

Comment:
When we use revisions of a project as we tend to use the commit date as version and append `-unstable` at the end of the pname.